Script/item.dat

```diff
@@ -1104,8 +1104,8 @@ potion /* materialcontainer-> */
   FormModifier = 25;
   NameSingular = "bottle";
   MainMaterialConfig == GLASS;
-  SecondaryMaterialConfig = { 11, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID; }
-  MaterialConfigChances = { 11, 75, 50, 5, 50, 1, 10, 5, 25, 5, 5, 25; }
+  SecondaryMaterialConfig = { 13, WATER, HEALING_LIQUID, OMMEL_URINE, POISON_LIQUID, VALDEMAR, ANTIDOTE_LIQUID, VODKA, TROLL_BLOOD, OMMEL_SWEAT, OMMEL_TEARS, SULPHURIC_ACID, MUSTARD_GAS_LIQUID, LIQUID_DARKNESS; }
```

Comment:
MUSTARD_GAS_LIQUID and LIQUID_DARKNESS should be removed from the spawn possibility. Players will flip when they see bottles spawning these materials because it means that is one less bottle of Ommel fluids or healing liquid. It's bad enough with those pesky hedgehogs slurping everything off the dungeon floor.